source/gloperate-glkernel/include/gloperate-glkernel/stages/MultiFrameAggregationPipeline.h

```diff
@@ -24,11 +32,11 @@ class MultiFrameAggregationStage;
 *  @brief
 *    Pipeline that aggregates multiple frames rendered by the given Stage/Pipeline
 */
-class GLOPERATE_API MultiFrameAggregationPipeline : public Pipeline
+class GLOPERATE_GLKERNEL_API MultiFrameAggregationPipeline : public gloperate::Pipeline
 {
 public:
     CPPEXPOSE_DECLARE_COMPONENT(
-        MultiFrameAggregationPipeline, gloperate::Stage
+        gloperate_glkernel::MultiFrameAggregationPipeline, gloperate::Stage
```

Comment:
You are inside the gloperate-glkernel namespace. There is no for a fully qualified class name.